base/base_inner.ts

```diff
@@ -1,3 +1,5 @@
+  const EventEmitter: typeof import('events').EventEmitter;
```

Comment:
```suggestion
  const NodeEventEmitter: typeof import('events').EventEmitter;
```

Can we make it obvious that this _is not_ an Electron specific EventEmitter rather it is node's event emitter

Reply:
I think your suggestion is better. I'll update.